### DISCUSSION TITLE
Build multiple tables from json array

### DISCUSSION THREAD

**Discussion by mamta** [mapfish/mapfish-print#2873] [General]
Hello,

As I mailing-list(https://groups.google.com/forum/#!forum/mapfish-print-users) appears to be down, I am posting my problem here.

I am trying to build a report with multiple tables similar to (https://github.com/mapfish/mapfish-print/tree/master/examples/src/test/resources/examples/datasource_dynamic_tables) using json datasource.

I can successfully build a table with json data in report_table example -
https://github.com/mamta/mapfishExamples/tree/main/report_table

Now I am trying to build multiple tables similar to datasource_dynamic_tables (https://github.com/mapfish/mapfish-print/tree/master/examples/src/test/resources/examples/datasource_dynamic_tables) using json array in report_table_multi example -
https://github.com/mamta/mapfishExamples/tree/main/report_table_multi

I need some advice for configuring processors -
https://github.com/mamta/mapfishExamples/blob/main/report_table_multi/config.yaml
Is it a right approach?

Thanks for consideration.
Beat regards.
Mamta

**Comment by sbrunner** [comment]
Yes it looks good :-)

**Reply by mamta** [reply]
Could you suggest how to configure processor in this case?
